federation/manifests/federation-etcd-pvc.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{.FEDERATION_APISERVER_DEPLOYMENT_NAME}}-etcd-claim
+  annotations:
+    volume.alpha.kubernetes.io/storage-class: "{{.PD_PLUGIN_NAME}}"
```

Comment:
Aah, now I understand.  I would have guessed that there would be a default for this per cloud provider, but I guess it's still in alpha, so perhaps the answer is "not yet".